types/jasmine/index.d.ts

```diff
@@ -837,12 +837,12 @@ declare namespace jasmine {
     }
 
     interface CustomReporter {
-        jasmineStarted?(suiteInfo: SuiteInfo): void;
-        suiteStarted?(result: CustomReporterResult): void;
-        specStarted?(result: CustomReporterResult): void;
-        specDone?(result: CustomReporterResult): void;
-        suiteDone?(result: CustomReporterResult): void;
-        jasmineDone?(runDetails: RunDetails): void;
+        jasmineStarted?(suiteInfo: SuiteInfo, done?: () => void): void | Promise<void>;
```

Comment:
I'm not sure if there is a way to overload interface methods.

I tried to do:

```js
jasmineStarted?(suiteInfo: SuiteInfo): void | Promise<void>;
jasmineStarted?(suiteInfo: SuiteInfo, done: () => void): void;
```

but it gave me an error.

Reply:
I think this is fine for now.